src/components/base-types/prop-type-validators.js

```diff
@@ -0,0 +1,16 @@
+
+const PropTypeValidator = {
+  arrayOf: (length, type = 'number') => {
+    return (propValue, key, componentName, location, propFullName) => {
+      if (propValue.length !== length) {
```

Comment:
Please, use `Array.isArray()` instead of checking the `length`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
I don't need to check if `propValue` is an array (I already know it). I want to check if a number of elements (length) equal a given value.